emails/2015/feb-meetup-call-for-speakers.md

```diff
@@ -0,0 +1,92 @@
+
+Ideas on finding a talk:
+
+1. Fork a talk
+
+Many, many talks have been given in the past, so instead of starting from the ground up, you could "fork" somebody else's talk and save a bunch of time, while still bringing lots of benefit to us. With a simple email, it's likely that you could get the permission of the original author to use their slides, code, etc.
+
+Places to find talks:
+
+Djangocon 2014: https://www.youtube.com/watch?v=cqP758k1BaQ&list=PLE7tQUdRKcybbNiuhLcc3h6WzmZGVBMr3
+Djangocon 2013: https://www.youtube.com/watch?v=drWhGUdadGw&list=PLtqtTJ4wP09YOFqm_lBCoQtmS6S0omW3J
+Djangocon Europe 2014: http://eventifier.com/event/djangoconeuropw/videos?page=2
+Search SpeakerDeck for Django: https://speakerdeck.com/search?utf8=%E2%9C%93&q=django (this brings up 44 pages of talks!)
+
+2. Get an idea / content for a talk from a blog post
+
+The Django and Python world are full or really smart and generous people. Start with what they have written, and (ask first), and turn it into a talk. A good place to look is on the archives of popular newletters that aggregate Django and Python articles:
+
+The "Django Round Up" by Lincoln Loop is excellent: http://us2.campaign-archive1.com/home/?u=24fce4628ba1d5814debef334&id=2d6dd01daf
+Python Weekly archives: http://www.pythonweekly.com/archive/
+pycoder's archives: http://pycoders.com/archive/
+
+3. If that doesn't work, here's a list of ideas that we recently created as "potential interesting talks"
+
+- Django with Haystack/Elasticsearch
+- Django and Docker
+- Django and AngularJS
+- Flask vs Django
+- Getting a dev environment set up with Django + Vagrant + Virtualbox
+- Django for a (Rails / PHP / .NET / etc) programmer
+- High Performance Django (a talk that uses the recent book by Lincoln Loop to discuss making Django fast)
+- Testing in Django (Given many times. Never too many)
+- What's up with security in Django? (what do I need to be worried about? What does Django take care of for me)
+- Django Rest Framework
+- The state of real-time applications with Django (what are my options?)
+- Getting the most out of the Django ORM
+
+4. Grab a friend, and give a talk together
+
+Talks don't have to be one person. Two people can tag-team and bring double the power to share their knowledge.
+
+5. Volunteer to give a talk, but ask for help
+
+If you want to give a talk, but don't think you're ready, or have the full time, email us anyway, and we will try to find somebody more experienced to guide you.
+
+6. Volunteer to help somebody prepare a talk
+
+If you don't have enough time to give a full talk, but have good knowledge in something, let us know, and maybe somebody will do most of the leg-work while you guide them.
+
+## Call for video help
+
+If you have video equipment and skills, we would love to hear from you. You can help us live-stream the talks, and record them to upload to our channels, allowing the speakers contribution to live on FOREVER!
+
```

Comment:
Cool. I'll leave this in for now, and we can also announce at the meetup on the 16th